src/main/java/mate/spring/into/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package mate.spring.into.service.impl;
+
+import java.util.List;
+import mate.spring.into.model.Book;
+import mate.spring.into.repository.BookRepository;
+import mate.spring.into.service.BookService;
+
+public class BookServiceImpl implements BookService {
+    private BookRepository bookRepository;
```

Comment:
How does it work without implementation?

Reply:
Read about dependency injection

Reply:
the approach is correct, but please consider using @RequiredArgsConstructor.